src/pymor/models/iosys.py

```diff
@@ -647,7 +678,10 @@ def hsv(self, mu=None):
         sv
             One-dimensional |NumPy array| of singular values.
         """
-        return self._hsv_U_V(mu=mu)[0]
+        if "hsv" in self.presets and mu is None:
```

Comment:
It's probably better to do the check in `__init__`, something like

```python
assert all(not obj.parametric for obj in [A, B, C, D, E])
```

before checking the properties of `presets`.